src/components/button/Button.tsx

```diff
@@ -82,6 +82,7 @@ class Button extends React.Component<ButtonProps> {
                 'is-disabled': isDisabled,
                 'is-loading': isLoading,
                 'is-selected': isSelected,
+                'bdl-is-disabled': isDisabled,
```

Comment:
Can you elaborate further on why we need a second `is-disabled` classname?

Reply:
I checked the commit log, and found this.
https://github.com/swfree/box-ui-elements/pull/2

A "bdl-" namespace was added to solve the conflicting classnames.
There's a [script](https://github.com/box/box-ui-elements/blob/master/scripts/utils/bdlClassnameManager) for this, but it doesn't cover all the case.